Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Meadow.Hardware;
+
+namespace Meadow.Foundation.Sensors.Atmospheric
+{
+    /// <summary>
+    /// BME680 Temperature, Pressure, Humidity and gas sensor
+    /// </summary>
+    /// <remarks>
+    /// This class implements the functionality necessary to read the temperature, pressure, humidity and VOS
+    /// </remarks>
+    namespace Meadow.Foundation.Sensors.Atmospheric
```

Comment:
Why encapsulate this in another namespace (which is the same name)?

Reply:
yeah that was copy/paste mistake - I'm shocked it compiles - I'll fix :)